### DISCUSSION TITLE
How to Sort and Hide (Page) in Navigation

### DISCUSSION THREAD

**Discussion by nurbaitifitriyani** [mecha-cms/mecha#204] [Q&A]
### Sortir Navigasi

Menu navigasi utama bawaan Mecha ditampilkan urut sesuai abjad. Saya ingin menyortir urutan menu navigasi secara manual, apakah bisa dilakukan eksekusi di fail `sort.data` atau adakah cara simpel yang bisa ditempuh?

 ### Menyembunyikan Laman dari Navigasi

Untuk menyembunyikan sebuah laman dari navigasi, apakah bisa dilakukan di dalam fail `state.data`?

**Comment by taufik-nurrohman** [comment]
Coba cek diskusi ini: https://github.com/mecha-cms/mecha/discussions/122

Untuk sortir manual mungkin bisa diakali dengan menambah properti `stack` sebagai referensi.

~~~ php
->sort([1, 'stack']);
~~~

Paling ideal sebenarnya kalau bisa dibuat secara terpisah [seperti ini](https://github.com/mecha-cms/mecha/discussions/148), cuma Saya masih belum punya ide finalnya.
